### ISSUE TITLE
Possible VS Code extension..?

### ISSUE THREAD

**Issue by GriffinG1** [leovoel/embed-visualizer#50]
Hi! Just wondering if you've considered adapting this into a VS Code extension? Would definitely be useful for seeing what your output will look like w/o needing to switch to discord, restart bot/reload module, and use a command.

Related note: is this even being actively maintained?

**Comment by barraIhsan** [comment]
Yes, it's pretty useful when maybe you use [Rest Client](https://github.com/Huachao/vscode-restclient) Extension to send HTTP requests (webhook). And then, there is the preview! Is a great combination
